src/udev/test-udev-rules-util.c

```diff
@@ -0,0 +1,60 @@
+#include <stdlib.h>
+#include <string.h>
+
+#include "macro.h"
+#include "string-util.h"
+#include "udev-rules-util.h"
+
+
+static void test_parse_value(void) {
+        // input: "valid operand"
+        // parsed: valid operand
+        // use the following command to help generate textual C strings:
+        // python3 -c 'import json; print(json.dumps(input()))'
```

Comment:
doesn't matter much, but we we generally want to use `//` C++ style comments only for "FIXME" style comments, and C-style `/* */` comments for explanatory stuf. hence please do so here too. See https://systemd.io/CODING_STYLE/#formatting